### PR TITLE
add Persian language (file:docs/README.fa.md) 🇮🇷

### PR DESCRIPTION
**Pull request by KomeilParseh**
#459

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,8 @@
 	<a href="./LICENSE.md"><img src="https://img.shields.io/badge/license-MIT-blue.svg"></a>
 </p>
 
-<p align="center">[<a href="./docs/README.ja.md">日本語</a>]</p>
+<p align="center">[<a href="https://github.com/o2sh/onefetch/docs/README.ja.md">日本語</a>, 
+<a href="https://github.com/o2sh/onefetch/docs/README.fa.md">فارسی</a>]</p>
```
Comment (spenserblack) [inline README.md:13]:
```suggestion
<p align="center">[<a href="https://github.com/o2sh/onefetch/docs/README.ja.md">日本語</a> | 
<a href="https://github.com/o2sh/onefetch/docs/README.fa.md">فارسی</a>]</p>
```
I think that `|` is a bit more common as a separator between languages. For example [naive-ui's README](https://github.com/TuSimple/naive-ui/blob/84f74a8f801ac93305efdc73c1e8962cfc081b48/README.md)

File: docs/README.fa.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+<p align="center">
+	<a href="https://crates.io/crates/onefetch"><img src="https://img.shields.io/crates/v/onefetch.svg" alt="cargo"></a>
+	<a href="https://github.com/o2sh/onefetch/actions"><img src="https://github.com/o2sh/onefetch/workflows/CI/badge.svg" alt="Build Status"></a>
+	<a href="https://github.com/o2sh/onefetch/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22"><img src="https://img.shields.io/github/issues/o2sh/onefetch/help%20wanted?color=green" alt="help wanted"></a>
+	<a href="./LICENSE.md"><img src="https://img.shields.io/badge/license-MIT-blue.svg"></a>
+</p>
+
+<p align="center">[<a href="https://github.com/o2sh/onefetch/docs/README.ja.md">日本語</a>, 
+<a href="https://github.com/o2sh/onefetch/docs/README.fa.md">فارسی</a>]</p>
```
Comment (spenserblack) [inline docs/README.fa.md:15]:
```suggestion
```
I don't think we're including links to the other READMEs in anything but the `README.md`.

For example, these aren't in `README.ja.md`.

**Review by spenserblack — CHANGES_REQUESTED** [review]
Thank you!

I left a couple of suggested changes.

Also, it looks like there is a change to the mode of `tools/image_to_ascii.py` that isn't needed (`100755 → 100644`). You might want to check that out.

**Comment by KomeilParseh** [comment]
Excellent, I will do it right now and send it.

**Comment by KomeilParseh** [comment]
> Thank you!
> 
> I left a couple of suggested changes.
> 
> Also, it looks like there is a change to the mode of `tools/image_to_ascii.py` that isn't needed (`100755 → 100644`). You might want to check that out.

Sorry I did not understand the second part, please explain more simply.😅🤦🏻‍♂️

**Comment by spenserblack** [comment]
>Sorry I did not understand the second part, please explain more simply.

No problem!

1. Go to "Files changed" tab
2. scroll down to `tools/image_to_ascii.py`
3. You should see something like "100755 → 100644"

For some reason the file mode changed, but I'm not sure why (possibly Windows OS). Then you probably used `git add .` or `git add --all`. This caused the change to file mode to be added.

**Comment by spenserblack** [comment]
Rebased to fix file mode

**Review by spenserblack — APPROVED** [review]

**Comment by spenserblack** [comment]
To learn more about file modes, you can look up `chmod`. Somehow the equivalent of `chmod -x tools/image_to_ascii.py` happened.

File: README.md [hunk 3]
```diff
@@ -9,7 +9,9 @@
 	<a href="./LICENSE.md"><img src="https://img.shields.io/badge/license-MIT-blue.svg"></a>
 </p>
 
-<p align="center">[<a href="./docs/README.ja.md">日本語</a>]</p>
+<p align="center">[<a href="https://github.com/o2sh/onefetch/docs/README.ja.md">日本語</a>
+|
+<a href="https://github.com/o2sh/onefetch/docs/README.fa.md">فارسی</a>]</p>
```
Comment (o2sh) [inline README.md:14]:
```suggestion
<p align="center">
    [<a href="docs/README.ja.md">日本語</a>] • [<a href="docs/README.fa.md">فارسی</a>]
</p>
```
We may want to use relative links here.

**Review by o2sh — APPROVED** [review]

**Comment by KomeilParseh** [comment]
> > Sorry I did not ...
> 1. Go to "Files changed" tab
> 2. scroll down to `tools/image_to_ascii.py`
> 3. You should see something like "100755 → 100644"

Hi @spenserblack 


Thanks, probably the last commit of this file deleted a few extra things or combined a few things.

If this script works then it does not matter!

**Comment by o2sh** [comment]
Question: This may be a small detail but I've just noticed that your translation doesn't contain the words "Git" nor "Rust"  - which are proper nouns. Is it it voluntary ?

**Comment by KomeilParseh** [comment]
> Question: This may be a small detail but I've just noticed that your translation doesn't contain the words "Git" nor "Rust" - which are proper nouns. Is it it voluntary ?


Hi, if I found out, you mean I did not write git and rust? If yes, because my editor code when rtl language like Persian makes it difficult to write next to an ltl language like English I wrote these two things in Persian rust:راست
git:گیت

Of course, because راست(right) in Persian means something else, I wrote زبان برنامه‌نویسی راست(rust programming language).

**Pull request merged**